src/components/shared/basic/ReleaseProcess/OfferPage/index.tsx

```diff
@@ -86,11 +87,11 @@ export default function OfferPage({
 
   useEffect(() => {
     if (fetchServiceStatus) dispatch(setServiceStatus(fetchServiceStatus))
-    fetchServiceStatus?.serviceTypeIds.every((item) =>
-      ['CONSULTANCE_SERVICE']?.includes(item)
+    fetchServiceStatus?.serviceTypeIds.includes(
+      ServiceTypeIdsEnum.DATASPACE_SERVICE
     )
-      ? skipTechnicalIntegrationStep(true)
-      : skipTechnicalIntegrationStep(false)
+      ? skipTechnicalIntegrationStep(false)
+      : skipTechnicalIntegrationStep(true)
```

Comment:
we can write this better. 
Instead of

`condition ? doSomething(false) : do Something(true)`

write

`doSomething(!condition)`